packages/composite-checkout/src/lib/payment-methods/existing-credit-card.js

```diff
@@ -89,12 +89,14 @@ function formatDate( cardExpiry ) {
 export function ExistingCardLabel( { last4, cardExpiry, cardholderName, brand } ) {
 	const { __ } = useI18n();
 
+	const maskedCardDetails = sprintf( __( '**** %s' ), last4 );
```

Comment:
This might benefit from using `_x` instead to give some context since it's sort of a cryptic string.